cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -56,6 +58,17 @@ def test_myservice_crud(appliance, setup_provider, context, order_catalog_item_i
         my_service.delete()
 
 
+@pytest.mark.meta(blockers=[BZ(1518056, forced_streams=["5.7"])])
+@pytest.mark.parametrize('context', [ViaSSUI])
+def test_retire_service(appliance, setup_provider, context, order_catalog_item_in_ops_ui):
+    """Tests retire service"""
+    service_name = order_catalog_item_in_ops_ui.name
+    with appliance.context.use(context):
+        appliance.server.login()
```

Comment:
**Question**
Is this `login` necessary, the navigation in `my_service.retire()` should handle navigation within the context?